MainGame.py

```diff
@@ -15,31 +19,53 @@ def PrepareGame(self) -> None:
         self.playerTwo.name = self.GetUserName("Spieler 2")
         self.playerTwo.symbol = "O"
 
-    def Turn(self, player: Player) -> None:
+    def Turn(self, player: Player) -> Optional[Player]:
         self.board.ShowBoard()
 
         while True:
             row: int = self.GetRowInput(player.name)
 
-            isvalidTurn: bool = self.board.IsValidTurn(row)
+            isValidTurn: bool = self.board.IsValidTurn(row)
 
-            if isvalidTurn:
+            if isValidTurn:
                 self.board.AddCoinToBoard(row, player.symbol)
-                return
 
-            print(
-                f"Du kannst deinen Stein nicht in Spalte '{row}' setzen. Versuch es nochmal!"
-            )
+                playerWins = StatusValidator.isWin(self.board, player, row)
+
+                if playerWins:
+                    return player
+                return player
+
+            else:
+                print(
+                    f"Du kannst deinen Stein nicht in Spalte '{row}' setzen. Versuch es nochmal!"
+                )
+
+                return player
 
-    def Round(self) -> None:
+    def Round(self) -> Optional[Player]:
         self.roundNumber += 1
+        if self.roundNumber > MAXROUNDS:
+            return Player()
         print(f"Round: {self.roundNumber}")
-        self.Turn(self.playerOne)
-        self.Turn(self.playerTwo)
+        winner: Player = self.Turn(self.playerOne)
```

Comment:
```suggestion
        player = self.Turn(self.playerOne)
```
Because the method returns `Player`, the type of `winner` is inferred. Also, `player` is a more fitting name :)